### ISSUE TITLE
Clarity and API for flushing denormal values

### ISSUE THREAD

**Issue by dashesy** [microsoft/onnxruntime#5117]
**Is your feature request related to a problem? Please describe.**
Does ONNXRuntime flush denormal values?
Please add that information to the documentation and, if not, provide an API to flush them

**System information**
- ONNX Runtime version (you are using): 1.4

**Describe the solution you'd like**
With weights too small, the denormal values hurt the performance. 
In PyTorch `torch.set_flush_denormal(true)` makes sure FTZ is called.

**Describe alternatives you've considered**
Manually trim the weights.

**Additional context**
Similar issue #4816 ask

**Comment by RyanUnderhill** [comment]
I don't think we do anything special for denormal values currently, but if you have any suggestions based on our code please let us know. Marking it as an enhancement.

**Comment by dashesy** [comment]
For CPU EP I think it makes sense to flush to zero by default.
It affects the performance considerably when slight change in weight decay could result in denormals.
